sections/semantics-textlevel.include

```diff
@@ -2702,17 +2702,48 @@
     <dd>Uses {{HTMLElement}}.</dd>
   </dl>
 
-  The <{u}> element <a>represents</a> a span of text with an unarticulated, though
-  explicitly rendered, non-textual annotation, such as labeling the text as being a proper name in
-  Chinese text (a Chinese proper name mark), or labeling the text as being misspelt.
-
+  The <{u}> element <a>represents</a> a span of text with a non-textual annotation,
+  such as labeling a proper name in Chinese text, a surname, or text as being misspelt.
+  
+  <div class="example">
+  Underlining as a stylistic presentation is a convention in some cases:
+  
+  <xmp highlight="html">
+    <p>Among others, Japanese, Hungarian and Chinese names are generally written with the family name first: <u>Wu</u> Xiaoqian.</p>
+  </xmp>
+  
+  <xmp highlight="html">
+    and when the sun rises over the trees of fire, there are no other <u class="spelling">incediaris</u> as powerful in this collection of dwelling
+  </xmp>
+  
+  <xmp highlight="html">
+    <p>The <u>see</u> is full of fish</p>
+  <xmp>
+  </div>
+  
   In most cases, another element is likely to be more appropriate: for marking stress emphasis,
-  the <{em}> element should be used; for marking key words or phrases either the
+  the <{em}> element
+  should be used; for marking key words or phrases either the
   <{b}> element or the <{mark}> element should be used, depending on the context;
   for marking book titles, the <{cite}> element should be used; for labeling text with explicit textual annotations, the
   <{ruby}> element should be used; for technical terms, taxonomic designation,
   transliteration, a thought, or for labeling ship names in Western texts, the <{i}>
   element should be used.
+  
+  <div class="example">
+  
+  Note that the examples above, and even more the following
+  
+    <xmp>
+    <abbr>HTML</abbr> stands for <u>H</u>yper<u>t</u>ext <u>M</u>arkup <u>L</u>anguage.
+    </xmp>
+
+  could and probably should be written
```

Comment:
I think, on reflection, because I was tired more than anything. I'll strip this bit out, and wait until we get some clarity in #1247 before playing with it again.